csharp/src/Ice/SSL/TransceiverI.cs

```diff
@@ -483,6 +498,7 @@ private int getSendPacketSize(int length) =>
     public int getRecvPacketSize(int length) =>
         _maxRecvPacketSize > 0 ? Math.Min(length, _maxRecvPacketSize) : length;
 
+    private string _errorDescription = "";
```

Comment:
I think it would be cleaner to use a nullable string:
```
private string? _errorDescription;
```

Reply:
I would enable nullable for this file in a follow-up PR.